### ISSUE TITLE
2-3 sec sound loss at track start when pc is connected to AV Receiver

### ISSUE THREAD

**Issue by CountZ3ro** [digimezzo/dopamine-windows#690]
## I'm submitting a...

<pre><code>
[ ] Regression 
[x ] Bug report  
[ ] Feature request
</code></pre>

## I'm using these versions

* Windows version: 10 Education, version 1079, Build 16299.192
* Dopamine version:1.5.10

## Reproduction steps

* Step 1 Connect PC that contains music files to AV receiver through HDMI (Tried on Onkyo TX-NR609)
* Step 2 Play a music file through Dopamine
* Step 3

## Observed behavior

Upon starting a music file or skipping to the next track, the song counter starts but the sound comes through the speakers with a 2 sec delay, missing always the beginning of each song. Restarting the song does not have this problem.
Other software such as Winamp don't have this problem on the same computer/setup.
Using the computer's built-in speakers instead of external receiver/speakers doesn't have this issue either.
Changing the Lag value in options has no effect.
Every time a song starts, the Receiver screen flashes as if the input was changed and sound eventually comes through after 2-3 secs (receiver connects to the source).
The issue restricts the use of Dopamine in a home theater PC setup as it disrupts the listening experience especially for songs that have no dead time between them (e.g. live concerts).

## Desired behavior

Smooth transition between songs and instant reaction to sound reproduction from external speakers.

## Log file

[Dopamine.log](https://github.com/digimezzo/Dopamine/files/1799716/Dopamine.log)

**Comment by digimezzo** [comment]
I've heard that before. It seems to be typical some some type of receivers. Some software work around that issue by adding a 2 seconds gap before the track starts playing.

**Comment by CountZ3ro** [comment]
Yes, but the problem exists only when using Dopamine. In Winamp for example or MP it doesn't happen. It has to do with the playing engine of the software I guess. It's like it disengages the sound card interrupting the signal to the receiver every time there is a gap in sound from a file stopping or ending making it search for input. Not sure what other software do differently...
